### ISSUE TITLE
fail to exporting to onnx

### ISSUE THREAD

**Issue by edwin191** [Yzichen/FlashOCC#98]
when running convert_bevdet_to_TRT.py ,

![Image](https://github.com/user-attachments/assets/2efbd3f7-86ba-4542-91e4-182d6bbaf1c0)

The model can be trained successfully, but when exporting to onnx error occur.

here is my pip list:
Package                  Version     Editable project location
------------------------ ----------- ---------------------------------
absl-py                  2.2.1
addict                   2.4.0
black                    24.8.0
cachetools               5.5.2
ccimport                 0.4.4
certifi                  2025.1.31
charset-normalizer       3.4.1
click                    8.1.8
coloredlogs              15.0.1
contourpy                1.1.1
cumm                     0.4.11
cycler                   0.12.1
Cython                   0.29.24
decorator                5.2.1
descartes                1.1.0
dill                     0.4.0
exceptiongroup           1.2.2
fire                     0.7.0
flake8                   7.1.2
flashocc_plugin          0.0.0       /zjk_ws/FlashOCC/projects
flatbuffers              25.2.10
fonttools                4.56.0
google-auth              2.38.0
google-auth-oauthlib     1.0.0
grpcio                   1.70.0
h5py                     3.11.0
humanfriendly            10.0
idna                     3.10
imageio                  2.35.1
importlib_metadata       8.5.0
importlib_resources      6.4.5
iniconfig                2.1.0
joblib                   1.4.2
kiwisolver               1.4.7
lark                     1.2.2
lazy_loader              0.4
llvmlite                 0.36.0
lyft-dataset-sdk         0.0.8
Mako                     1.3.9
Markdown                 3.7
MarkupSafe               2.1.5
matplotlib               3.5.3
mccabe                   0.7.0
mmcls                    0.25.0
mmcv-full                1.5.3
mmdeploy                 0.9.0       /zjk_ws/ori_occ/FlashOCC/mmdeploy
mmdet                    2.25.1
mmdet3d                  1.0.0rc4    /zjk_ws/FlashOCC/mmdetection3d
mmsegmentation           0.25.0
mpmath                   1.3.0
multiprocess             0.70.18
mypy-extensions          1.0.0
narwhals                 1.33.0
networkx                 2.2
ninja                    1.11.1.4
numba                    0.53.0
numpy                    1.23.5
nuscenes-devkit          1.1.11
nvidia-cublas-cu11       11.10.3.66
nvidia-cuda-nvrtc-cu11   11.7.99
nvidia-cuda-runtime-cu11 11.7.99
nvidia-cuda-runtime-cu12 12.8.90
nvidia-cudnn-cu11        8.5.0.96
nvidia-tensorrt          8.4.1.5
oauthlib                 3.2.2
onnx                     1.17.0
onnxruntime-gpu          1.11.0
opencv-python            4.11.0.86
packaging                24.2
pandas                   2.0.3
pathspec                 0.12.1
pccm                     0.4.16
pillow                   10.4.0
pip                      24.2
platformdirs             4.3.6
plotly                   6.0.1
pluggy                   1.5.0
plyfile                  1.0.3
portalocker              3.0.0
prettytable              3.11.0
protobuf                 5.29.4
pyasn1                   0.6.1
pyasn1_modules           0.4.2
pybind11                 2.13.6
pycocotools              2.0.7
pycodestyle              2.12.1
pycuda                   2025.1
pyflakes                 3.2.0
pyparsing                3.1.4
pyquaternion             0.9.9
pytest                   8.3.5
python-dateutil          2.9.0.post0
pytools                  2024.1.14
pytz                     2025.2
PyWavelets               1.4.1
PyYAML                   6.0.2
requests                 2.32.3
requests-oauthlib        2.0.0
rsa                      4.9
scikit-image             0.19.3
scikit-learn             1.3.2
scipy                    1.10.1
setuptools               59.5.0
Shapely                  1.8.5.post1
six                      1.17.0
spconv                   2.3.6
sympy                    1.13.3
tensorboard              2.14.0
tensorboard-data-server  0.7.2
tensorrt                 10.9.0.34
tensorrt_cu12            10.9.0.34
tensorrt_cu12_bindings   10.9.0.34
tensorrt_cu12_libs       10.9.0.34
termcolor                2.4.0
terminaltables           3.1.10
threadpoolctl            3.5.0
tifffile                 2023.7.10
tomli                    2.2.1
torch                    1.13.0
torchaudio               0.13.0
torchvision              0.14.0
tqdm                     4.67.1
trimesh                  2.35.39
typing_extensions        4.13.0
tzdata                   2025.2
urllib3                  2.2.3
vscodedebugvisualizer    0.2.0
wcwidth                  0.2.13
Werkzeug                 3.0.6
wheel                    0.44.0
yapf                     0.40.1
zipp                     3.20.2

can somebody help me fix it??

**Comment by heemang0826** [comment]
Hi did you solve the problem? I encountered same error as you, so I modified this code : projects/mmdet3d_plugin/ops/bev_pool_v2/bev_pool.py

Seems like it's caused by tensorrt version difference.
[bev_pool.py](https://github.com/user-attachments/files/26398881/bev_pool.py)
